chunk_utils.py

```
@@ -0,0 +1,209 @@
+        # Create the output directory if it doesn't exist
+        chunk_dir = Path(output_dir)
+        chunk_dir.mkdir(parents=True, exist_ok=True)
+        
+        if output_format == "jsonl":
+            # Save all chunks to a single JSONL file
+            output_file = chunk_dir / "chunks.jsonl"
+            with open(output_file, "w", encoding="utf-8") as f:
+                for chunk in chunks:
+                    f.write(json.dumps(asdict(chunk)) + "\n")
+            return
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)